### ISSUE TITLE
Facet by nationality

### ISSUE THREAD

**Issue by escowles** [pulibrary/orangelight#1925]
Use cases:
> Volker Schröder teaches the art and culture of 17th century France, emphasizing the prints created under Louis XIV. These artists all created images of French architecture, landscape, and design (650) but were themselves from Amsterdam, Rome, and London along with Paris. Without nationality, there is no way to study the influx of artists to Paris at this time.
>
>In the early years of the United States, books printed in Boston or New York used plates (illustrations) that were printed in London and Paris. Without nationality, there is no way to know the printmakers were separate from the publishers.
>
> Many fine art prints are reproductions of fine art paintings, such as J.B. Jackson’s “Miracle of St. Mark” [left sheet; Le Bl 15, Kainen 23]. Jackson was a British artist who went to Venice to create these prints and then back to London to market them. Without nationality, we have no way of searching this history as the 650 is only Italian.

**Comment by christinach** [comment]
@escowles do we still want to implement this feature or can I close it?

**Comment by escowles** [comment]
@christinach Yes, please keep this issue open. Julie Melby has been asking for this for a long time.

**Comment by christinach** [comment]
ok!

**Comment by hectorcorrea** [comment]
We already capture the place of publication, but it seems that we will also need the author/printmaker's nationality which we currently don't capture, is that a correct assumption? 

If that is the case, where would we fetch from the nationality of the author/printmaker?

**Comment by christinach** [comment]
We would need to ask the catalogers about that.

**Comment by mzelesky** [comment]
The Library of Congress Name Authority File has all kinds of information on authors. There was a plan way back when to index authorities in Blacklight. Without that, I think this effort would be quite difficult.

**Comment by mzelesky** [comment]
We made significant progress on indexing MARC authority records and other work around authorities, but this effort was postponed in 2019.

**Comment by escowles** [comment]
👍🏻 Yes, I was expecting artist nationality to come from the authority records. I don't know if that makes this a good small pilot project for indexing authority data, or if that means there is a lot of infrastructure to setup before this would be practical.

**Comment by mzelesky** [comment]
I think there is a lot of infrastructure to set up. Mike Gibney at Penn developed a really elegant way of having authority 'See' references, but it required a specific version of different modules. Most likely, there would have to be a separate index/search interface for authority records, then from a Name search, you could link to results in Orangelight.

**Comment by hectorcorrea** [comment]
Some of the authorities work that Mark alluded to exists in these branches:

* https://github.com/pulibrary/orangelight/tree/authority-vocab
* https://github.com/pulibrary/orangelight/tree/authority-vocab-bl7

...and this other project: https://github.com/pulibrary/authority_control

**Comment by caroldh** [comment]
The larger project related to indexing authority files would be a good feature and high value, but this particular aspect on its own is not pressing.
